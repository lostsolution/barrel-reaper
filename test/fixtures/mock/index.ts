import type { InterfaceA, InterfaceB } from '@barrelreaper/test/fixtures/barrel';
import {
    Enum,
    ModuleA,
    ModuleB,
    type TypeA,
    type TypeB,
    a1 as a1LocalRenamed,
    a2,
    a3,
    b1 as b1LocallyRenamed,
    b2,
    b3,
    default as def,
    enumValue,
} from '@barrelreaper/test/fixtures/barrel';
import { b1Renamed, b2Renamed } from '@barrelreaper/test/fixtures/barrel/index';

void def;
void a1LocalRenamed;
void a2;
void a3;
void b1LocallyRenamed;
void b2;
void b3;
void b2Renamed;
void b1Renamed;
void ModuleA.a1;
void ModuleB.b1;

export type A = TypeA;
export interface IA extends InterfaceA {}

export type B = TypeB;
export interface IB extends InterfaceB {}

void enumValue;
void Enum;
