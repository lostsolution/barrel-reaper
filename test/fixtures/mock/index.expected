import type { InterfaceA } from '@barrelreaper/test/fixtures/barrel/module-a';
import type { InterfaceB } from '@barrelreaper/test/fixtures/barrel/module-b';
import { Enum } from '@barrelreaper/test/fixtures/barrel/module-enum';
import { ModuleA } from '@barrelreaper/test/fixtures/barrel/module-a';
import { ModuleB } from '@barrelreaper/test/fixtures/barrel/module-b';
import type { TypeA } from '@barrelreaper/test/fixtures/barrel/module-a';
import type { TypeB } from '@barrelreaper/test/fixtures/barrel/module-b';
import { a1 as a1LocalRenamed } from '@barrelreaper/test/fixtures/barrel/module-a';
import { a2 } from '@barrelreaper/test/fixtures/barrel/module-a';
import { a3 } from '@barrelreaper/test/fixtures/barrel/module-a';
import { b1 as b1LocallyRenamed } from '@barrelreaper/test/fixtures/barrel/module-b';
import { b2 } from '@barrelreaper/test/fixtures/barrel/module-b';
import { b3 } from '@barrelreaper/test/fixtures/barrel/module-b';
import def from '@barrelreaper/test/fixtures/barrel/index';
import { enumValue } from '@barrelreaper/test/fixtures/barrel/module-enum';
import { b1Renamed } from '@barrelreaper/test/fixtures/barrel/module-b';
import { b2Renamed } from '@barrelreaper/test/fixtures/barrel/module-b';

void def;
void a1LocalRenamed;
void a2;
void a3;
void b1LocallyRenamed;
void b2;
void b3;
void b2Renamed;
void b1Renamed;
void ModuleA.a1;
void ModuleB.b1;

export type A = TypeA;
export interface IA extends InterfaceA {}

export type B = TypeB;
export interface IB extends InterfaceB {}

void enumValue;
void Enum;
